src/ConnectedMode/Migration/IMigrationPrompt.cs

```diff
@@ -18,13 +18,22 @@
  * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  */
 
+using System;
 using System.ComponentModel.Composition;
+using Microsoft.VisualStudio.Shell;
+using Microsoft.VisualStudio.Shell.Interop;
+using SonarLint.VisualStudio.Core;
+using SonarLint.VisualStudio.Core.Notifications;
+using System.Threading.Tasks;
 
 namespace SonarLint.VisualStudio.ConnectedMode.Migration
 {
+    /// <summary>
+    /// Controls a gold bar that prompts the user to initiate migration.
```

Comment:
My bad, used the word gold bar again. Should I change it to "In control of showing / clearing a migration prompt" ?